ScoopBackup/Lib.ps1

```diff
@@ -16,7 +17,7 @@ function Get-Buckets {
 function Get-Apps([string] $appsDir) {
     $apps = @{}
     Get-ChildItem $appsDir | ForEach-Object {
-        $current = "$_\current"
+        $current = "$($_.FullName)\current"
```

Comment:
Other changes are nice, But I don't understand this row. Is this necessary?

Reply:
When I was running it, I was not in the scoop directory and Get-Apps was returning nothing, turned out it was because it was giving a relative path I believe.

I was surprised as well as I thought FullName was the default string representation.  I could double-check if you like.

Reply:
Just tested, I may have jumped to that conclusion too quickly.  I was, at the time, trying to make it work on default PowerShell as well but eventually gave up and installed PSCore 7. I don't do much w/ powershell usually.

I pushed out a reversion of this line now.